packages/palette/src/elements/Cards/MediumCard.ios.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react"
+import { Image, View } from "react-native"
+import LinearGradient from "react-native-linear-gradient"
+import { color, space } from "../../helpers"
+import { Flex } from "../Flex"
+import { Spacer } from "../Spacer"
+import { Sans } from "../Typography"
+import { Tag } from "./CardTag.ios"
+import { MediumCardProps } from "./MediumCard.shared"
+
+/**
+ * `MediumCard` is a card with one image one tall image, and text for title and subtitle
+ * at the bottom.
+ */
+export const MediumCard: React.FC<MediumCardProps> = ({
+  image,
+  title,
+  subtitle,
+  tag,
+}) => {
+  return (
+    <View
+      style={{
+        width: "100%",
+        aspectRatio: 1.0 / 1.33,
+        flexDirection: "row",
+        borderRadius: 4,
+        overflow: "hidden",
+      }}
+    >
+      <Flex flex={2} background={color("black10")}>
+        <Image source={{ uri: image }} style={{ flex: 1 }} />
+      </Flex>
+      <LinearGradient
+        colors={["rgba(0, 0, 0, 0)", "rgba(0, 0, 0, 1)"]}
+        style={{
+          position: "absolute",
+          width: "100%",
+          height: "100%",
+          opacity: 0.15,
+        }}
+      />
+      <Flex
+        style={{
+          position: "absolute",
+          bottom: 0,
+          left: space(1),
+          right: space(6),
+        }}
+      >
+        <Sans size="5t" color={color("white100")}>
+          {title}
+        </Sans>
+        {!!subtitle && (
+          <Sans size="3t" color={color("white100")}>
+            {subtitle}
+          </Sans>
+        )}
+        <Spacer mt={1} />
+      </Flex>
+      {!!tag && (
+        <Tag
+          {...tag}
+          style={{ position: "absolute", top: space(1), left: space(1) }}
```

Comment:
I think all the padding in this file should be `15` rather than `space(1)`

Reply:
For these ones we talked with tin and he said instead of 15 it's ok to use the `space(1)`, since we don't have a 15 in our regular usage.

Reply:
Ok, but for future reference we use 15px in quite a few places in eigen because in those places 10px looks too small and 20px looks too big :) I don't know whether this is one of those places, but IMO the spacing system is more like guide rails than hard rules and it's fine to do custom spacing when it _looks_ more 'right' than any of the predefined options.

Reply:
also `15px` is just `0.5` + `1` in our design system values :p